external/slowparse/slowparse.js

```diff
@@ -1382,6 +1392,12 @@
     },
 
     // This is a helper function to determine whether a given string
+    // is an obsolete HTML element tag
+    _knownObsoleteHTMLElement: function(tagName) {
+      return this.obsoleteHtmlElements.indexOf(tagName) > -1;
```

Comment:
indexOf(...) !== -1 is more standard.  Also, it's not really worth making a helper function that's one line especially when it's only used in one place.